### PR TITLE
Adjust general validator util

### PR DESCRIPTION
**Pull request by orlandohohmeier**
Align the validator util name with the other validator utils and remove the redundant methods.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @aldipower

File: src/js/utils/Validator.js [hunk 1]
```diff
@@ -1,32 +0,0 @@
-var Validator = {
-  isEmail: function (email) {
-    return email != null &&
-      email.length > 0 &&
-      !/\s/.test(email) &&
-      /.+@.+\..+/
-      .test(email);
-  },
-
-  hasNoWhitespaces: function (str) {
```
Comment (aldipower) [inline src/js/utils/Validator.js]: So, these function are not needed anymore, correct?

**Comment by aldipower** [comment]
Cool, code got removed. I like this. :)
LGTM

File: src/js/utils/Validator.js (hunk 1 above)
Comment (orlandohohmeier) [inline src/js/utils/Validator.js]: Yep, we don't need them anymore. :)

**Comment by pierlo-upitup** [comment]
retest this please

**Comment by pierlo-upitup** [comment]
🚢

**Pull request merged**